fhir-model/src/test/java/com/ibm/fhir/model/path/test/FHIRPathAsTest.java

```diff
@@ -33,9 +35,14 @@ void testAsOperation() throws Exception {
                                  .build();
 
         FHIRPathEvaluator evaluator = FHIRPathEvaluator.evaluator();
-        Collection<FHIRPathNode> result = evaluator.evaluate(patient, "Patient.deceased as dateTime");
-        
-        assertEquals(result.size(), 0, "Number of selected nodes");
+        try {
+//          Collection<FHIRPathNode> result = evaluator.evaluate(patient, "Patient.deceased as dateTime");
+            evaluator.evaluate(patient, "Patient.deceased as dateTime");
+            fail();
+        } catch (FHIRPathException e) {
+            
+        }
+//      assertEquals(result.size(), 0, "Number of selected nodes");
```

Comment:
What does this test expect as an outcome? The assertEquals comment and use of fail() are confusing my understanding ... 

if the intent is to throw an Exception, add an assertTrue(true, "expected to reach here");

Reply:
I changed this back to the original code after relaxing the `as` operator behavior.